StyleCop.Analyzers/StyleCop.Analyzers.Test/LayoutRules/SA1516UnitTests.cs

```diff
@@ -0,0 +1,260 @@
+﻿using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.CodeAnalysis.CodeFixes;
+using Microsoft.CodeAnalysis.Diagnostics;
+using Xunit;
+using StyleCop.Analyzers.LayoutRules;
+using TestHelper;
+
+namespace StyleCop.Analyzers.Test.LayoutRules
+{
+    public class SA1516UnitTests : CodeFixVerifier
+    {
+        private const string DiagnosticId = SA1516ElementsMustBeSeparatedByBlankLine.DiagnosticId;
+
+        private const string CorrectCode = @"extern alias Foo1;
```

Comment:
:question: Does the reference StyleCop implementation require a blank line between the `extern alias` directive and the using alias directives?